src/Sylius/Bundle/PromotionBundle/Resources/config/services.xml

```diff
@@ -51,6 +52,7 @@
         <parameter key="sylius.form.transformer.promotion_coupon_to_code.class">Sylius\Bundle\PromotionBundle\Form\DataTransformer\CouponToCodeTransformer</parameter>
 
         <parameter key="sylius.validator.date_range.class">Sylius\Bundle\PromotionBundle\Validator\PromotionDateRangeValidator</parameter>
+        <parameter key="sylius.validator.generation_amount.class">Sylius\Bundle\PromotionBundle\Validator\CouponGenerationAmountValidator</parameter>
```

Comment:
Service id should be more specific. ``sylius.validation.promotion_coupon_generation_amount``.